compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.cpp

```diff
@@ -68,34 +68,39 @@ static void addBufferizePasses(OpPassManager &passManager) {
   BufferizationOptions::MemCpyFn memcpyFn = gpuCopyFn;
   addIREEComprehensiveBufferizePasses(passManager, allocationFn, deallocationFn,
                                       memcpyFn);
+  passManager.addPass(createCanonicalizerPass());
+  passManager.addPass(createCSEPass());
 }
 
-static void tileAndBufferize(OpPassManager &pm) {
+static void tileAndDistributeToWorkgroup(OpPassManager &pm) {
   pm.addPass(createTileAndDistributeToWorkgroupsPass());
 
   auto &nestedModulePM = pm.nest<ModuleOp>();
   nestedModulePM.addNestedPass<func::FuncOp>(
       createConvertToDestinationPassingStylePass());
   nestedModulePM.addPass(createCanonicalizerPass());
   nestedModulePM.addPass(createCSEPass());
+}
 
-  addBufferizePasses(nestedModulePM);
+static void tileAndBufferize(OpPassManager &pm) {
```

Comment:
DCE this ?

Reply:
This is still used line 114, the goal is to remove that everywhere but this is only the first step.